examples/tutorials/02_cozmo_face/01_cozmo_face_image.py

```diff
@@ -36,16 +36,12 @@ def get_in_position(robot):
             robot.set_head_angle(cozmo.robot.MAX_HEAD_ANGLE).wait_for_completed()
 
 
-def run(sdk_conn):
-    '''The run method runs once Cozmo is connected.'''
-
-    robot = sdk_conn.wait_for_robot()
-
+def cozmo_program(robot: cozmo.robot.Robot):
     get_in_position(robot)
 
     # load some images and convert them for display cozmo's face
-    image_settings = [("images/cozmosdk.png", Image.BICUBIC),
-                      ("images/hello_world.png", Image.NEAREST)]
+    image_settings = [("../../face_images/cozmosdk.png", Image.BICUBIC),
+                      ("../../face_images/hello_world.png", Image.NEAREST)]
```

Comment:
we should at least have a `lib`  folder at the top level that can hold things like `flask_helpers.py`.  Any programs that import packages/modules specific to that example can then have their own `lib` (or `common` as i guess we used already) folder too within their program directory, so that we continue to keep runnable programs out of package directories.